lib/fs.js

```diff
@@ -572,17 +571,53 @@ function openSync(path, flags, mode) {
  * @param {{
  *   type?: string;
  *   }} [options]
- * @returns {Promise<Blob>}
+ * @param {(
+ *   err?: Error,
+ *   blob?: Blob
+ *   ) => any} callback
```

Comment:
```suggestion
 *   ) => void} callback
```

Reply:
Isn't this user-provided callback? Its return value is ignored, but it's still `any`.